remote.go

```diff
@@ -844,6 +844,14 @@ func (wd *remoteWD) SetAlertText(text string) error {
 }
 
 func (wd *remoteWD) execScriptRaw(script string, args []interface{}, suffix string) ([]byte, error) {
+	for index, value := range args {
+		if ori, ok := value.(*remoteWE); ok {
+			args[index] = ori.toMap()
+		} else if ori, ok := value.(remoteWE); ok {
+			args[index] = ori.toMap()
+		}
+	}
```

Comment:
Please add a test. The test should demonstrate that some trivial script can make effective use of a WebElement passed in this way.